frontend/src/pages/Alerts/LogAlert/LogAlertPage.tsx

```diff
@@ -134,16 +176,13 @@ export const LogAlertPage = () => {
 	const header = (
 		<Box
 			display="flex"
-			justifyContent="space-between"
+			justifyContent="flex-end"
 			alignItems="center"
 			borderBottom="dividerWeak"
 			px="8"
 			py="6"
 			cssClass={styles.header}
 		>
-			<Text userSelect="none">
-				{capitalize(createStr)} log monitoring alert
-			</Text>
 			<Box display="flex" alignItems="center" gap="4">
 				<Button
 					kind="secondary"
```

Comment:
would be good if it dynamically went back to either the alerts page or the logs page depending on where you came from originally

Reply:
Maybe we could use `Link`'s `state` prop to pass some context here that would allow you to conditionally go back to the `/logs` if we know that's where you came from.